addons/hr_timesheet/controllers/portal.py

```diff
@@ -128,12 +128,12 @@ def portal_my_timesheets(self, page=1, sortby=None, filterby=None, search=None,
 
         def get_timesheets():
             field = None if groupby == 'none' else groupby
-            orderby = '%s, %s' % (field, sortby) if field else sortby
+            orderby = f'{field}, {sortby}' if field and field != sortby.split(' ')[0] else sortby
```

Comment:
What's the point of this change? Is it to avoid having the same field twice in the order by? I don't think that's a problem as far as I know

Reply:
Looking back, I also see that this is unnecessary. The 'orderby' would get "date, date" and I thought this was related to the wrong outputs while I was debugging and trying to understand how it worked. Sorry for the trouble!